experimental/federated_plugin/federated_plugin/example/pubspec.yaml

```diff
@@ -0,0 +1,23 @@
+name: federated_plugin_example
+description: Demonstrates how to use the federated_plugin plugin.
+
+publish_to: 'none' # Remove this line if you wish to publish to pub.dev
+
+environment:
+  sdk: ">=2.7.0 <3.0.0"
+
+dependencies:
+  flutter:
+    sdk: flutter
+
+  federated_plugin:
+    path: ../
+  cupertino_icons: ^0.1.3
+
+dev_dependencies:
+  flutter_test:
+    sdk: flutter
+  pedantic: ^1.9.0
+
+flutter:
+  uses-material-design: true
```

Comment:
nit: Put a newline on this. If the file was autogenerated without one, we should file a bug.

Reply:
The error was from my side while removing the comments.